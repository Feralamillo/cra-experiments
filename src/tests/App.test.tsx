import React from 'react';
import { shallow } from 'enzyme';

import { App } from '../App';

jest.mock('../components/header', () => ({
  Header: 'mock-header',
}));
jest.mock('../components/main', () => ({
  Main: 'mock-main',
}));
jest.mock('../components/footer', () => ({
  Footer: 'mock-footer',
}));

test('renders the component', () => {
  const wrapper = shallow(<App />);

  expect(wrapper).toMatchSnapshot();
});
