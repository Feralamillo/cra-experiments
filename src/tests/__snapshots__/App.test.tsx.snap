// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<Fragment>
  <mock-header />
  <mock-main />
  <mock-footer />
</Fragment>
`;
